README.md

```diff
@@ -1,88 +1,52 @@
-				},
-				"body": {
-					"key": "can be exported as json, if the Content-Type is json",
-					"next": "or just string if json is not applicable"
-				}
-			}
-			"response": {
-				"body": {
-					"access_token": "also can be exported as json structure, or pure string"
-				},
-				"duration": number_of_ms,
-				"header": {
-					"Content-Type": "application/json",
-					"Content-Length": "5152",
-					"...": "..."
-				},
-				"code": 200,
-				"status": "OK"
-			},
-			"assertions": [
-				{
-					"name": "Status code is 200",
-					"skipped": false,
-					"failed": false,
-				},
-				{
-					"name": "Token is valid",
-					"skipped": false,
-					"failed": true,
-					"errorMessage": "expect.fail()"
-				}
-			]
-		},
-		{
-			"name": "name of the second step",
-			"request": "another request structure",
-			"reponse": "another response structure",
-			"assertions": "another array of tests"
-		},
-		{
-			"...": "..."
-		}
-	],
+  "testExecutionKey": "WRFL-123",
+  "info": {
+    "project": "WRFL",
+    "summary": "Execution of automated tests",
+    "description": "This execution is automatically created when importing execution results from Gitlab",
+    "version": "",
+    "revision": "",
+    "startDate": "2023-07-05T06:18:04.815-05:30",
+    "finishDate": "2023-07-05T06:18:10.603-05:30",
+    "testPlanKey": "",
+    "testEnvironments": []
+  },
+  "tests": [
+    {
+      "testKey": "WRFL-144",
+      "comments": "[WRFL-144] Step 3: Retrieving Workflow Modules with no matching filters - Assertion - Status code is 200 FAILED. Detail - expected response to have status code 201 but got 200\n[WRFL-144] Step 3: Retrieving Workflow Modules with no matching filters - Assertion - Should contain not contain any Workflow modules data FAILED. Detail - expected 0 to equal 1",
+      "status": "FAILED",
+      "start": "2023-07-05T06:18:10.606-05:30",
+      "finish": "2023-07-05T06:18:10.607-05:30",
+      "executedBy": "",
+      "assignee": "",
+      "defects": [],
+      "evidence": [],
+      "customFields": []
+    }
+  ]
 }
 ```
 
 [Structures described.](https://github.com/postmanlabs/newman#newmanruncallbackerror-object--summary-object)
 
 ## Options
 
-Option | Value | Optional | Note
--- | -- | --
---reporter-json-steps-export | <path/to/generate/json/report> | Yes |
---reporter-json-steps-stats  | | Yes | Include also stats into report
+| Option                                  | Value                        | Required | Note                          |
+|-----------------------------------------|------------------------------|----------|-------------------------------|
+| --reporter-xray-json-export             | /path/to/save/json/file.json | YES      | Path to store JSON report     |
+| --reporter-xray-json-project-key        | WRFL                         | YES      | JIRA Project key              |
+| --reporter-xray-json-test-execution-key | WRFL-123                     | YES      | JIRA X-Ray Test Execution key |
```

Comment:
this is good but i would just change the names of the switches to make it more consistent with future where we can add more exporters if we want
This also means i leave the original switches intact and add more switches for xray and thereby when more exporters come up then i keep adding more switches which can then further be used to choose which createXYZSummary to call.